### PR TITLE
[O11y][WebSphere Application Server] Resolve the conflict in host.ip field

### LINKED ISSUE #7404: [O11y][WebSphere Application Server] Conflicts in host.ip field
In WebSphere Application Server, found that `host.ip` field has conflicts because it is not mentioned in the `ecs.yml`. Please refer to the below screenshot.

conflict

- [x] Need to add the `host.ip` field to the ecs.yml file.
- [x] Add data re-indexing steps to the README.

## Related issue

- https://github.com/elastic/integrations/issues/7281

### PR DESCRIPTION
**Pull request by aliabbas-elastic**
- Bug

## What does this PR do?

Added host.ip field in ecs.yml and added re-indexing steps to resolve conflicts for existing users.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.

## Related issues

- Closes #7404
- Relates #7281

### CODE REVIEW

**Review by harnish-elastic — APPROVED** [review]

File: packages/websphere_application_server/docs/README.md [hunk 1]
```diff
@@ -11,6 +11,103 @@ This integration uses Prometheus to collect above metrics.
 
 To open Prometheus endpoint read following [instructions](https://www.ibm.com/docs/en/was/9.0.5?topic=mosh-displaying-pmi-metrics-in-prometheus-format-metrics-app).
 
+### Troubleshooting
+
+Conflicts in any field in any data stream can be solved by reindexing the data.
```
Comment (muthu-mps) [inline packages/websphere_application_server/docs/README.md:16]: Can we update the description on when to perform reindexing?

Comment (aliabbas-elastic) [reply packages/websphere_application_server/docs/README.md]: Thanks @muthu-mps for pointing it out. I referenced this [PR](https://github.com/elastic/integrations/pull/7444) and it seems we don't need this line as the below one specifies when to perform reindexing. Let me know if that is what you suggested.

**Review by muthu-mps — APPROVED** [review]
Looks good!

**Review by muthu-mps — APPROVED** [review]
Looks good!

**Pull request merged**